src/bioclip/predict.py

```diff
@@ -162,10 +156,23 @@ def create_bioclip_tokenizer(tokenizer_str="ViT-B-16"):
 
 
 class BaseClassifier(object):
-    def __init__(self, device: Union[str, torch.device] = 'cpu', model_str: str = MODEL_STR):
+    def __init__(self, model_str: str, pretrained_str: str | None = None, device: Union[str, torch.device] = 'cpu'):
```

Comment:
I'm getting an error when creating either classifier without specifying model_str:
```
classifier = TreeOfLifeClassifier()
...
TypeError: BaseClassifier.__init__() missing 1 required positional argument: 'model_str'
```
This error also occurs in the tests.

If we give model_str a default value it will fixes the error:

```suggestion
    def __init__(self, model_str: str = MODEL_STR, pretrained_str: str | None = None, device: Union[str, torch.device] = 'cpu'):
```

Reply:
Ah yes, sorry, I forget the invocation directly as a library. Will fix.

Reply:
Should be fixed in [013bc36](https://github.com/Imageomics/pybioclip/pull/33/commits/013bc3611c7ecd96c8efb93e2550fcbcf3fc2e26)